### DISCUSSION TITLE
Reset to master from conflicting-master-XXXXX

### DISCUSSION THREAD

**Discussion by Strum355** [zeapo/Android-Password-Store#2298] [Q&A] [answered]
After resolving merge issues on the conflicting-master-XXXXX branch, the app does not seem to be able to track master again. Instead, its still stuck on conflicting-master-XXXX according to "Local Git config & utilities". It is not clear to me whether this is expected or not (the branch name seems to imply not)

**Comment by msfjarvis** [comment]
What version of the app are you on?

**Comment by Strum355** [comment, marked as answer]
1.13.5 from the Play Store I believe

**Reply by msfjarvis** [reply]
I can't remember if the stable build is supposed to have a UI for it but the crude solution for it is to use the 'Delete repository' option in settings and then clone the repo again, provided you've resolved the conflicts on the remote and there's nothing on the device that has not been synced out. I've made a note to improve the user experience around this for the v2.0.0 release.

**Reply by Strum355** [reply]
Was hoping i wouldnt have to resolve to that, the option does exist so ill just do that then. 
Edit: Cant mark your reply as the answer
